### ISSUE TITLE
Pass Context to Template

### ISSUE THREAD

**Issue by PeterCasa** [mkeller1992/ngx-tooltip-directives#1]
Hi,

would it be possible to pass context to the tooltip template?

Something like (modified from your documentation):
<ng-template #myTemplate let-text="text">
  <div style="color: blue; font-weight: bold;">
   {{ text }}
  </div>
</ng-template>

**Comment by mkeller1992** [comment]
I can introduce the property `[tooltipContext]`. Usage would be as follows:

```ts
context = { estimate: 10 };
```

```html
<ng-template #myTemplate let-numberOfLessons="estimate">
    {{ numberOfLessons }} lessons
</ng-template>

<div [tooltipTemplate]="myTemplate"
     [tooltipContext]="context">
     Show Tooltip Template with Context
</div>
```

Would this meet your needs?

**Comment by PeterCasa** [comment]
That would be great, thank you!

**Comment by mkeller1992** [comment]
I included the new feature in version 19.1.0 which I just published. Can you please check if it's working fine?
